Source/4_0/PeripheralType.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+import CoreBluetooth
+import RxSwift
+
+protocol PeripheralType {
```

Comment:
We need to think what will be available for PeripheralType, and what will be available for ConnectedPeripheralType